src/deviation.jl

```diff
@@ -137,7 +137,7 @@ end
 
 Return the mean absolute deviation between two arrays: `mean(abs(a - b))`.
 """
-meanad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = L1dist(a, b) / length(a)
+meanad(a::AbstractArray{T}, b::AbstractArray{T}) where {T<:Number} = L1dist(a, b) / length(a)
```

Comment:
Yeah, I noticed that too. I mentioned it because the new syntax is slightly longer in that case, which makes this line go beyond the 92 char threshold.